packages/react-preferences/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@trycourier/react-preferences",
+  "version": "0.0.1-dev",
+  "main": "dist/index.js",
+  "types": "typings/index.d.ts",
+  "scripts": {
+    "babel": "babel src -d dist --extensions \".ts,.tsx\" --ignore \"src/**/__tests__/**\"",
+    "build:watch": "yarn babel --watch",
+    "build": "rimraf dist && yarn babel",
+    "clean": "rimraf dist",
+    "test": "jest -c jest.config.js --runInBand --silent",
+    "type-check": "tsc --noEmit",
```

Comment:
missing "readme" script

Reply:
could we add readme when after we are #69?